src/plugins/intel_cpu/src/nodes/mkldnn_def_conv_node.cpp

```diff
@@ -822,8 +821,8 @@ void MKLDNNDeformableConvolutionNode::DefConvExecutor::prepareSamplingWeights(
                     lh = (h_high < cur_h_end ? lh : 0);
                     lw = (w_high < cur_w_end ? lw : 0);
 
-                    const int h_off_low = h_ind_low * srcStrides[2] / srcStrides[3];
-                    const int h_off_high = h_ind_high * srcStrides[2] / srcStrides[3];
+                    const int h_off_low = h_ind_low * (srcStrides[2] / srcStrides[3]);
+                    const int h_off_high = h_ind_high * (srcStrides[2] / srcStrides[3]);
```

Comment:
Why we need this change?

Reply:
We have here `int X = A * B / C` expression. `A` of type `int`, `B`, `C` of type `size_t == ulong`. If `A >= 0`, everything is ok. In some cases A becames negative. So, at first expression casts to size_t (but still correct in `int` representation). But after 'unsigned' division we have got incorrect result, and segfault afterwards. 
In 'A * (B / C)` case the result of division is the same in `int` and `size_t` representation, then we have `*` operation with casting to `size_t` and casting to `int`. The result in every step is equivalent for both types. So, the general idea, to prevent division with differently represented operands.

Reply:
We have `int X = A * B / C` expression. A of type `int`, B, C of type 'size_t == unsigned long int`. If A >= 0, everything is ok. In some cases A < 0. Then `A * B` was casted to `size_t`, but still remains correct in `int` representation. But after 'unsigned' division the result becames incorrect in signed representation. The role of parenthesis is preventing of division of differently represented operands.

Reply:
Example:
-1 (int) * 102 (size_t) / 3 (size_t)
`A * B / C`
A * B = 18446744073709551514 (size_t)
A * B / C = 1431655732 (size_t) = 1431655732 (int)
`A * (B / C)`
B / C = 34
A * (B / C) = - 34 (int)